nkms/keystore/keystore.py

```diff
@@ -78,7 +85,7 @@ def get_key(self, fingerprint: bytes) -> Union[keypairs.EncryptingKeypair,
             key = txn.get(fingerprint)
 
         if not key:
-            return None
+            raise KeyNotFound("No key found with the provided fingerprint.")
```

Comment:
A better message here is, "No key with fingerprint {fingerprint}" - don't you think?

Reply:
Yeah, I'll throw that in.